Python/OTP_verifier.py

```diff
@@ -0,0 +1,15 @@
+import random
+from twilio.rest import Client
+
+otp=random.randint(1000,9999)
+print(otp)
+account_sid='Find from your twilio account'
```

Comment:
Keep this in a `.env` file and use a module like `dotenv` to import them in your Scipt. It makes it easy for us to track the Environment Variables.